### ISSUE TITLE
bug: `ion-toggle` and rendered label blocks vertical gestures/touch-scrolling

### ISSUE THREAD

**Issue by lincolnthree** [ionic-team/ionic-framework#27254]
### Prerequisites

- [X] I have read the [Contributing Guidelines](https://github.com/ionic-team/ionic-framework/blob/main/.github/CONTRIBUTING.md#creating-an-issue).
- [X] I agree to follow the [Code of Conduct](https://ionicframework.com/code-of-conduct).
- [X] I have searched for [existing issues](https://github.com/ionic-team/ionic-framework/issues) that already report this problem, without success.

### Ionic Framework Version

v7.x

### Current Behavior

With the new 'modern form syntax' `ion-toggle` now renders or contains its label as a child element. This means that its gesture config is now taking over the entire width of the parent container, and blocks touch/scroll gestures on not just the toggle icon/button, but on the entire label/row. This can lead to some very confusing user interactions.

See video:

https://user-images.githubusercontent.com/362329/233424827-c6d2ed5a-8492-4736-b211-47a5ca038eea.mov

Note, I have not tried it, but I suspect that other elements such as `ion-range`, which also have gestures, may also suffer from this same issue.

### Expected Behavior

The toggle's label should not be a gesture capturing element.

### Steps to Reproduce

Checkout repro url and scroll as seen in video.

### Code Reproduction URL

https://github.com/lincolnthree/ionic-bugs/tree/issue-27254

### Ionic Info

Ionic:

   Ionic CLI                     : 6.20.3 (/Users/lincoln/.nvm/versions/node/v16.14.0/lib/node_modules/@ionic/cli)
   Ionic Framework               : not installed
   @angular-devkit/build-angular : 15.1.6
   @angular-devkit/schematics    : 15.1.6
   @angular/cli                  : 15.1.6
   @ionic/angular-toolkit        : 6.1.0

Capacitor:

   Capacitor CLI      : 4.6.3
   @capacitor/android : not installed
   @capacitor/core    : 4.6.3
   @capacitor/ios     : not installed

Utility:

   cordova-res                          : not installed globally
   native-run (update available: 1.7.2) : 1.7.1

System:

   NodeJS : v16.14.0 (/Users/lincoln/.nvm/versions/node/v16.14.0/bin/node)
   npm    : 8.3.1
   OS     : macOS

**Comment by liamdebeasi** [comment]
Thanks! Mind giving this dev build a shot?

```
7.0.4-dev.11682008337.11c1bc0d
```

**Comment by lincolnthree** [comment]
Hey Liam,

This does not seem to resolve my issue. I'm going to try a few more times, but as of right now, I have not noticed a change in gestures when interacting with toggles.

```
Ionic:

   Ionic CLI                     : 7.0.1 (/Users/lincoln/.nvm/versions/node/v16.14.0/lib/node_modules/@ionic/cli)
   Ionic Framework               : @ionic/angular 7.0.4-dev.11682008337.11c1bc0d
   @angular-devkit/build-angular : 15.1.6
   @angular-devkit/schematics    : 15.1.6
   @angular/cli                  : 15.1.6
   @ionic/angular-toolkit        : 6.1.0
```

**Comment by liamdebeasi** [comment]
Make sure `@ionic/core@7.0.4-dev.11682008337.11c1bc0d` is getting installed as that is where the fix is.

**Comment by lincolnthree** [comment]
> Make sure `@ionic/core@7.0.4-dev.11682008337.11c1bc0d` is getting installed too as that is where the fix is.

![image](https://user-images.githubusercontent.com/362329/233664669-ee370857-be6b-43af-9f49-23f0b3cdb99d.png)

**Comment by lincolnthree** [comment]
I tried both in my production app and also in the sample repo.

**Comment by lincolnthree** [comment]
Let me try a full reinstall of node modules.

**Comment by liamdebeasi** [comment]
Good catch! Here's a dev build with another fix: `7.0.4-dev.11682092361.1656526f`

**Comment by lincolnthree** [comment]
Thanks @liamdebeasi - The latest Dev build appears to fix the issue!
